charts/pre-api/values.yaml

```diff
@@ -23,9 +23,17 @@ java:
         - name: apim-sub-smoketest-secondary-key
           alias: APIM_SUB_SMOKETEST_SECONDARY_KEY
         - name: app-insights-connection-string
+        - name: pp-clientid
+          alias: AZURE_CLIENT_ID
+        - name: pp-clientsecret
+          alias: AZURE_CLIENT_SECRET
   environment:
     RUN_DB_MIGRATION_ON_STARTUP: false
     APIM_ENABLED: true
     APIM_URI: https://sds-api-mgmt.platform.hmcts.net
+    AZURE_TENANT_ID: 531ff96d-0ae9-462a-8d2d-bec7c0b42082
+    AZURE_SUBSCRIPTION_ID: 74dacd4f-a248-45bb-a2f0-af700dc4cf68
```

Comment:
You'll want to make sure these are appropriately overridden in other envs. Having these staging values in this file means that by default everything will use staging. May be safter to put them in values.staging.template.yaml. Either way, be sure to add the correct ones for use here too please: https://github.com/hmcts/sds-flux-config/blob/master/apps/pre/pre-portal/prod.yaml
Give me a shout if you need a hand